src/cast.c

```diff
@@ -1386,6 +1386,31 @@ Type *toStaticArrayType(SliceExp *e)
 }
 
 /* ==================== castTo ====================== */
+/**
+ * This function should be called by iterateAliasThis.
+ * It tries to substitute a next subtype expression and check
```

Comment:
It will return false and will not write anything to `*outexpr`.
The similar logic can be applied to any `iterateAliasThis` delegates.